src/app/shared/components/locale-switcher/locale-switcher.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Inject, LOCALE_ID } from '@angular/core'
+
+@Component({
+  selector: 'sofie-locale-switcher',
+  templateUrl: './locale-switcher.component.html',
+  styleUrls: ['./locale-switcher.component.scss'],
+})
+export class LocaleSwitcherComponent {
+  public locales = [
+    { code: 'en-US', name: 'EN' },
+    { code: 'da', name: 'DK' },
+  ]
+
+  constructor(@Inject(LOCALE_ID) public activeLocale: string) {}
+
+  public onChange(): void {
+    window.location.href = `/${this.activeLocale}`
```

Comment:
Shouldn't it redirect to the current page in the new locale?

Reply:
After some discussions with you and Lindved, I have decided to remove the locale-switcher feature, since we anyway wont be able to use it for now.